specification/communication/data-plane/Microsoft.CommunicationServicesChat/preview/2020-11-01-preview3/communicationserviceschat.json

```diff
@@ -0,0 +1,1675 @@
+        "topic"
+      ],
+      "type": "object",
+      "properties": {
+        "topic": {
+          "description": "The chat thread topic.",
+          "type": "string",
+          "example": "Lunch Thread"
+        },
+        "participants": {
+          "description": "Participants to be added to the chat thread.",
+          "type": "array",
+          "items": {
+            "$ref": "#/definitions/ChatParticipant"
+          }
+        }
+      }
+    },
+    "ChatThread": {
+      "description": "Chat thread.",
+      "type": "object",
+      "properties": {
+        "id": {
+          "description": "Chat thread id.",
+          "type": "string",
+          "readOnly": true,
+          "example": "19:uni01_uy5ucb66ugp3lrhe7pxso6xx4hsmm3dl6eyjfefv2n6x3rrurpea@thread.v2"
+        },
+        "topic": {
+          "description": "Chat thread topic.",
+          "type": "string",
+          "example": "Lunch Chat thread"
+        },
+        "createdOn": {
+          "format": "date-time",
+          "description": "The timestamp when the chat thread was created. The timestamp is in RFC3339 format: `yyyy-MM-ddTHH:mm:ssZ`.",
+          "type": "string",
+          "readOnly": true,
+          "example": "2020-10-30T10:50:50Z"
+        },
+        "createdBy": {
+          "description": "Id of the chat thread owner.",
+          "type": "string",
+          "readOnly": true,
+          "example": "8:acs:8540c0de-899f-5cce-acb5-3ec493af3800_0e59221d-0c1d-46ae-9544-c963ce56c10b"
+        },
+        "deletedOn": {
+          "format": "date-time",
+          "description": "The timestamp when the chat thread was deleted. The timestamp is in RFC3339 format: `yyyy-MM-ddTHH:mm:ssZ`.",
+          "type": "string",
+          "example": "2020-10-30T10:50:50Z"
+        }
+      }
+    },
+    "CreateChatThreadErrors": {
+      "description": "Errors encountered during the creation of the chat thread.",
+      "type": "object",
+      "properties": {
+        "invalidParticipants": {
+          "description": "The participants that failed to be added to the chat thread.",
+          "type": "array",
+          "items": {
+            "$ref": "#/definitions/Error"
+          },
+          "readOnly": true
+        }
+      }
+    },
+    "CreateChatThreadResult": {
+      "description": "Result of the create chat thread operation.",
+      "type": "object",
+      "properties": {
+        "chatThread": {
+          "$ref": "#/definitions/ChatThread"
+        },
+        "errors": {
+          "$ref": "#/definitions/CreateChatThreadErrors"
+        }
+      }
+    },
+    "ChatThreadInfo": {
+      "description": "Summary information of a chat thread.",
+      "type": "object",
+      "properties": {
+        "id": {
+          "description": "Chat thread id.",
+          "type": "string",
+          "readOnly": true,
+          "example": "19:uni01_uy5ucb66ugp3lrhe7pxso6xx4hsmm3dl6eyjfefv2n6x3rrurpea@thread.v2"
+        },
+        "topic": {
```

Comment:
I think we should also do a pass and mark everything as x-nullable: false where applicable